src/components/admin/DeleteModal.tsx

```diff
@@ -33,5 +33,5 @@ export default observer(function DeleteModal({ mutationFn, translate, endpoint }
     deleteMutation.mutate(selectedRecord.id)
   }
 
-  return <DeleteDialog deleteHandler={deleteHandler} />
+  return <DeleteDialog modalStore={ModalStore} deleteHandler={deleteHandler} />
```

Comment:
Only in this location we use the exported const `ModalStore` from `'stores/dashboard/ModalStore'`

Do you think we need to migrate it so all components that use `<DeleteModal />`  pass their `modalStore` instance?

Reply:
Of course, I do.
I asked a question in our Discord group about that component. I don't see where it is used so I know where to define that instance.

Reply:
@dpmarkov I think you need to pass the modal store to `<DeleteModal />` the same way we pass it to `<DeleteDialog />` 

In this way whoever uses delete modal will need to provide their local instance of modal store.